run-dask-process.sh

```diff
@@ -114,12 +115,6 @@ function buildUCXWithInfinibandArgs {
 
     export UCX_MAX_RNDV_RAILS=1
     export UCX_MEMTYPE_REG_WHOLE_ALLOC_TYPES=cuda
-
-    export DASK_UCX__CUDA_COPY=True
-    export DASK_UCX__TCP=True
-    export DASK_UCX__NVLINK=True
-    export DASK_UCX__INFINIBAND=True
-    export DASK_UCX__RDMACM=True
     export DASK_RMM__POOL_SIZE=0.5GB
```

Comment:
Scheduler and client will also need `DASK_DISTRIBUTED__COMM__UCX__CREATE_CUDA_CONTEXT=True` here.